src/commandHandlers/bio.ts

```diff
@@ -44,11 +46,21 @@ export const command = async (arg: string, embed: MessageEmbed, message: Message
 
     // set information
     if (args[1]) {
-        const field = args[0].toLowerCase();
-        const data = args[1].toLowerCase();
+        const field = args[0].toLowerCase().trim();
```

Comment:
@eddiejaoude is this code a fix for the location stuff? has this just accidentally snuck in the code?

Reply:
yep I snuck the `location` in bio and also a few little fixes 😱